pkg/kubeapiserver/options/authentication.go

```diff
@@ -163,11 +163,6 @@ func (s *BuiltInAuthenticationOptions) WithWebHook() *BuiltInAuthenticationOptio
 	return s
 }
 
-func (s *BuiltInAuthenticationOptions) Validate() []error {
```

Comment:
Get it.
I need sometime to learn those flags.
Thanks very much for your suggestion.